src/ajax/viewlist.php

```diff
@@ -121,6 +121,16 @@
     die(AJAX_NO_AUTH);
 }
 
+// Load the columns to skip from the request. The external viewer uses this.
+$aColsToSkip = (!empty($_REQUEST['skip'])? $_REQUEST['skip'] : array());
+
+// Submitters should not be allowed to retrieve more information
+//  about users than the info the access sharing page gives them.
+if ($sObject == 'User' && $_AUTH['level'] < LEVEL_MANAGER) {
```

Comment:
@mkroon1 , this code is different from what it was in [3.0-20a](https://github.com/LOVDnl/LOVD3/blob/caaaace8c7c95b6354097a169a23fa5bea1eb3c5/src/ajax/viewlist.php#L119). There, it was a check on `LEVEL_SUBMITTER`. However, `LEVEL_CURATOR` is also not allowed to see more columns, right? I don't understand why it was `LEVEL_SUBMITTER`, do you know?

Reply:
From what I understand of the commit that introduced this check, it was made specifically to block submitters, since those had access to the new viewlist to designate colleagues. I guess collaborators were not considered at the time. I.e.: no specific reason why it didn't say `< LEVEL_MANAGER`.

Reply:
OK, cool! Collaborators are also kind of submitters, but not stopped by a check on `LEVEL_SUBMITTER`, so it's best to keep the `LEVEL_MANAGER` check then. Thanks!